src/components/MultiHopTrade/helpers.ts

```diff
@@ -67,3 +71,38 @@ export const getMixpanelEventData = () => {
     tradeType,
   }
 }
+
+type ProtocolFeeDisplay = {
+  assetId: AssetId | undefined
+  chainName: string | undefined
+  amountCryptoPrecision: string
+  symbol: string
+}
+
+export const parseAmountDisplayMeta = (items: AmountDisplayMeta[]): ProtocolFeeDisplay[] => {
+  const feeMap: Record<string, ProtocolFeeDisplay> = {}
+
+  items
+    .filter(({ amountCryptoBaseUnit }) => bnOrZero(amountCryptoBaseUnit).gt(0))
+    .forEach(({ amountCryptoBaseUnit, asset }: AmountDisplayMeta) => {
+      const key = `${asset.assetId}-${getChainAdapterManager()
+        .get(asset.chainId)
+        ?.getDisplayName()}`
```

Comment:
[suggestion]:
```suggestion
      const key = `${asset.assetId}-${chainIdToChainDisplayName(asset.chainId)}`
```

Reply:
Actually better still, this asset ID is specific to the chain so we can remove it without risk of key collisions:
```suggestion
      const key = asset.assetId
```

Reply:
https://github.com/shapeshift/web/pull/8462/commits/c05650223951f819f01ffd4d055c68d2adf7e212